server/src/main/webapp/WEB-INF/urlrewrite.xml

```diff
@@ -60,6 +60,20 @@
     <to last="true">/spark/api/pipelines/${escape:$1}/compare/${escape:$2}/${escape:$3}</to>
   </rule>
 
+  <rule>
+    <name>Pipeline Instance API</name>
+    <condition name="Accept">application\/vnd\.go\.cd\+json</condition>
+    <from>^/api/pipelines/([^/]+)/instance/([^/]+)(/?)$</from>
+    <to last="true">/spark/api/pipelines/${escape:$1}/instance/${escape:$2}</to>
+  </rule>
+
+  <rule>
+    <name>Pipeline Instance API</name>
+    <condition name="Accept">application\/vnd\.go\.cd\.v1\+json</condition>
+    <from>^/api/pipelines/([^/]+)/instance/([^/]+)(/?)$</from>
+    <to last="true">/spark/api/pipelines/${escape:$1}/instance/${escape:$2}</to>
+  </rule>
+
```

Comment:
Instead of using two different rules can we use only one rule and use regex in the condition?
`<condition name="Accept">application\/vnd\.go\.cd(.v*)*\+json</condition>` condition can be used instead of using 2 different rules with different conditions.